code/processes/tailsort.q

```diff
@@ -0,0 +1,77 @@
+\d .ts
+
+taildir:hsym `$getenv`KDBTAIL;                                             /-load in taildir env variables
+hdbdir:hsym `$getenv`KDBHDB;                                               /-load in hdb env variables
+savelist:@[value;`savelist;`quote`trade];                                  /-list of tables to save to HDB
+taildbs:key taildir;                                                       /-list of tailDBs that need saved to HDB
+taildirs:();                                                               /-empty list to append tailDB paths to - to be used
+                                                                           / when HDB save is complete to delete tailDB partitions
+\d .
+
+savescompleted:0;                                                          /-variable to count how many tailDBs have been saved to HDB
+
+endofday:{[pt;procname]
+  /- function to trigger data load & save to HDB once endofday message is received from tailer(s)
+  .lg.o[`endofday;"end of day message received from ",string[procname]," - ",string[pt]];
+  loadandsave[pt;procname];
+  };
+
+loadandsave:{[pt;procname]
```

Comment:
not just this function, but throughout this file convention would be to re-arrange so that functions are defined before they are used e.g. `loadandsave` is used inside `endofday`, so `loadandsave` should be defined above it in the file etc.

Reply:
okay yeah I'll change the order of functions in this file, thanks